src/mesos_state_client.erl

```diff
@@ -317,11 +317,19 @@ port_mapping(#{container_port := ContainerPort, host_port := HostPort, protocol
 force_pull_image(#{force_pull_image := ForcePullImage}) -> ForcePullImage;
 force_pull_image(_) -> false.
 
+docker_network(BinString) when is_binary(BinString) ->
+    String = binary_to_list(BinString),
+    case string:to_lower(String) of
+        "bridge" -> 'bridge';
+        "host" -> 'host';
+        "user" -> 'user'
+    end.
+
 docker(#{image := Image, network := Network} = Docker) ->
     #docker{
        force_pull_image = force_pull_image(Docker),
        image = Image,
-       network = list_to_existing_atom(string:to_lower(binary_to_list(Network))),
+       network = docker_network(Network),
```

Comment:
In the spirit of the rest of the function names in the module, it probably makes mores sense to use `network_type` since we're not actually parsing anything